resources/views/users/DispWords.blade.php

```diff
@@ -0,0 +1,35 @@
+@extends('layouts.app')
```

Comment:
In this file please change file name. Must use camelCase format for blade name not StudlyCase format